content/en/tracing/trace_collection/custom_instrumentation/java/dd-api.md

```diff
@@ -200,20 +200,24 @@ Add `@Trace` to methods to have them be traced when running with `dd-java-agent.
 
 Datadog's Trace annotation is provided by the [dd-trace-api dependency][6].
 
-`@Trace` annotations have the default operation name `trace.annotation` and resource name of the traced method. These can be set as arguments of the `@Trace` annotation to better reflect what is being instrumented. These are the only possible arguments that can be set for the `@Trace` annotation.
+The available arguments for the `@Trace` annotation are:
+
+- `operationName`: Set the operation name for the trace (default: The method's name).
+- `resourceName`: Set the resource name for the trace (default: The same value as `operationName`).
+- `noParent`: Set to `true` to always start a new trace at that method. Supported from v1.22.0+ of `dd-trace-java` (default: `false`).
 
 ```java
 import datadog.trace.api.Trace;
 
 public class SessionManager {
 
-    @Trace(operationName = "database.persist", resourceName = "SessionManager.saveSession")
+    @Trace(operationName = "database.persist", resourceName = "SessionManager.saveSession", noParent = true)
```

Comment:
```suggestion
    @Trace(operationName = "database.persist", resourceName = "SessionManager.saveSession")
```
Setting `noParent=true` is something we only advise when the customer wants to break up a long trace. For that reason I wouldn't update the example, otherwise customers might think this is the recommended value.